src/modules/Builder/features/Activities/ActivitiesHeader/ActivitiesHeader.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import { ActivitiesHeader } from '.';
+
+describe('ActivitiesHeader - Interaction Tests', () => {
+  test('calls onAddActivity when the Add Activity button is clicked', () => {
```

Comment:
could you pls add test case, that two buttons are present in the ActivitiesHeader component (add activity and add performance task)?

Reply:
It is not nes, There is no case when they are not  presented)